test/Microsoft.AspNetCore.Mvc.Core.Test/ModelBinding/Metadata/ModelAttributesTest.cs

```diff
@@ -188,10 +189,11 @@ public void GetAttributesForParameter_NoAttributes()
                     .GetParameters()[0]);
 
             // Assert
-            Assert.Empty(attributes.Attributes);
+            // Not exactly "no attributes" due to SerializableAttribute on object.
+            Assert.IsType<SerializableAttribute>(Assert.Single(attributes.Attributes));
```

Comment:
Which change causes us to start seeing this? I'm a little confused about that.

Reply:
Is this the impact of https://github.com/aspnet/Mvc/pull/7657/files#diff-e40550dccb02e5e7fd23ad2da6a23908R201

Reply:
Yes, the `SerializableAttribute` always existed but this test was introduced _after_ we began ignoring attributes on the parameter's type.